platform/integrations/ex-integration-guide/exclient-upload-recordings.js

```diff
@@ -47,6 +49,27 @@ async function uploadRecordings(fileName) {
         let response = await client.loginClientCredentialsGrant(clientId, clientSecret);
         logger.verbose('Login successfully');
 // >> START ex-get-presigned-url
+
+        var output = fs.createWriteStream(fileName);
+        var archive = archiver('zip', {
+            store: true,
+            // uploaded zip file must be uncompressed, setting compression level to 0
+            zlib: { level: 0 }
+        });
+
+        archive.on('error', function(err) {
+            throw err;
+        });
+
+        archive.pipe(output);
+
+        // append files
+        archive.file('./call1.opus.bin', {name: 'call1.opus.bin'});
```

Comment:
In the existing text on the guide page at https://developer.genesys.cloud/platform/integrations/ex-integration-guide/recording-injection-example we have "myfile.opus.bin" being used as the example, so let's use the same one here.